File: level1/C++/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>
using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    
    for(int i=0;i<commands.size(); i++){
        vector<int> ans;
        for(int j=commands[i][0]-1; j<commands[i][1];j++){
            ans.push_back(array[j]);
        }
        sort(ans.begin(), ans.end());
        // cout << ans[commands[i][2]-1];
        answer.push_back(ans[commands[i][2]-1]);
        
    }
    
    return answer;
}

File: level1/K번째 수.py
def solution(array, commands):
    answer = []

    for i in range(len(commands)):
        cpy_arr=array[commands[i][0]-1:commands[i][1]]
        cpy_arr.sort()
        answer.append(cpy_arr[commands[i][2]-1])
    return answer
